docs/faq.md

```diff
@@ -118,18 +136,57 @@ Yes, this is the plan - to use a HWW like Coldcard/Trezor with Specter DIY, with
 
 It gives you the ability to store the transaction temporarily before it is signed.
 
-## USAGE
+## *If the Bitcoin Core instance we are connecting to already has a wallet, is it possible to load it via the UI if we know the name, and could we import a .dat file?*
+
+Currently, you can create a hot wallet from within the specter-desktop UI, but at the moment it's not possible to extract XPUBs from the existing Core wallet, and without XPUBs change verification will break in all hardware wallets. Change address verification in multisig on a hardware wallet requires ability to check that change and input were derived from the same XPUBs. Without XPUBs all hardware wallets will show two outputs so you never know if the change output is actually change or not.
```

Comment:
> but at the moment it's not possible to extract XPUBs from the existing Core wallet

Wallets created by Bitcoin Core always use hardened derivations, so they don't have useful xpubs - this breaks multisig address verification on hardware wallets and they can't verify change addresses. Therefore Specter is creating a Bitcoin Core hot wallet differently - it generates a bip39 recovery phrase, loads xprvs to Core and xpubs to Specter. Then it can be used as a part of multisig setup as usual.